week_3/algorithms/case_insensitive_string_sorting/case_insensitive_string_sorting.cpp

```diff
@@ -0,0 +1,40 @@
+#include <algorithm>
+#include <cctype>
+#include <iostream>
+#include <string>
+#include <vector>
+
+bool Abs(const std::string& s1, const std::string& s2) {
+  char a, b;
+  for (char c : s1) {
+    a = tolower(c);
```

Comment:
@Ypostolov you are always comparing the last symbols in the string
Try experiment : acb abc
the output should be abc, acb but your algorithm will return acb abc

Reply:
Fixed.
Please check 5427d2d4744c69594af34708effc0a7d3a3570c2.